src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ImportCompletionItem.cs

```diff
@@ -16,7 +16,8 @@ namespace Microsoft.CodeAnalysis.Completion.Providers
 {
     internal static class ImportCompletionItem
     {
-        private const string SortTextFormat = "~{0} {1}";
+        public const char SortTextPrefix = '~';
+        private static readonly string s_sortTextFormat = $"{SortTextPrefix}{0} {1}";
```

Comment:
oh... oh wow... that's awful.  

What's going on here, and can we doc why we need this?

Reply:
this is particularly gnarly as it's basically just picking some random char and hoping that that's not something any roslyn language might ever use in an identifier.  imagine a world where `~` became a legal identifier character.  For example, in F# you can identifiers starting with `_`, `$`, `@`, `(grave accent)`, or `?`.  If we had picked any of those as this special character then that would break languages.  It's not a great idea to embed this sort of thing.

Reply:
i'm also super confused by this.  If we can already have a flag stating if it's an expanded item... then why don't we just compare that flag first (instead of extracting this information out from this string)?

Reply:
Fixed as you suggested.